### PR TITLE
Pin torchvision.

### PR DESCRIPTION
**Pull request by ailzhang**

### CODE REVIEW

File: .circleci/common.sh [hunk 1]
```diff
@@ -12,3 +12,16 @@ function clone_pytorch() {
...
+  # retry 3 times
+  # old versions of pip don't have the "--progress-bar" flag
+  pip install --progress-bar off "$@" || pip install --progress-bar off "$@" || pip install --progress-bar off "$@" ||\
+  pip install "$@" || pip install "$@" || pip install "$@"
+}
+
+function install_torchvision() {
+  # Check out torch/vision at Jun 11 2020 commit
+  # This hash must match one in .jenkins/caffe2/test.sh
+  pip_install --user git+https://github.com/pytorch/vision.git@c2e8a00885e68ae1200eb6440f540e181d9125de
```
Comment (dlibenzi) [inline .circleci/common.sh:26]: Can we fetch this from the one PT uses?

File: .circleci/common.sh [hunk 2]
```diff
@@ -12,3 +12,16 @@ function clone_pytorch() {
   git clone --quiet https://github.com/pytorch/pytorch.git "$PYTORCH_DIR"
   cp -r "$PWD" "$XLA_DIR"
 }
+
+function pip_install() {
+  # retry 3 times
+  # old versions of pip don't have the "--progress-bar" flag
+  pip install --progress-bar off "$@" || pip install --progress-bar off "$@" || pip install --progress-bar off "$@" ||\
```
Comment (dlibenzi) [inline .circleci/common.sh:19]: Really?!? 😄

Comment (ailzhang) [reply .circleci/common.sh:19]: lol it happens...

**Comment by ailzhang** [comment]
This has passed the point where master is failing, so merging to unblock master.
A followup can be found at https://github.com/pytorch/pytorch/pull/40828/files.

**Pull request merged**